src/helpers/messages/edit_message.ts

```diff
@@ -29,6 +29,12 @@ export async function editMessage(channelId: bigint, messageId: bigint, content:
     validateComponents(content.components);
   }
 
+  // TODO: v12 remove
+  if (content.embed) content.embeds = [content.embed, ...(content.embeds || [])];
```

Comment:
I think content.embed should be updated to be undefined after this

Reply:
Discord does not care about `embed` if we send an `embeds` too. It will always check for `embeds` first